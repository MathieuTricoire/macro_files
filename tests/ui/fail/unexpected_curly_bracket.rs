use macro_files::create;

fn main() {
    create!({
        "directory" {}
    });
}
