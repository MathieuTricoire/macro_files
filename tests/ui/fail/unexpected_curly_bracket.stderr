error: no rules expected the token `{`
 --> tests/ui/fail/unexpected_curly_bracket.rs:5:21
  |
5 |         "directory" {}
  |                     ^ no rules expected this token in macro call
